crates/ruff/src/registry.rs

```diff
@@ -491,6 +491,7 @@ ruff_macros::register_rules!(
     rules::flake8_pie::rules::UnnecessaryDictKwargs,
     rules::flake8_pie::rules::PreferListBuiltin,
     rules::flake8_pie::rules::SingleStartsEndsWith,
+    rules::flake8_pie::rules::PreferSimpleAnyAll,
```

Comment:
Our convention is that rules should be named such that "allow X" would be a sensical phrase. I guess the flake8-pie rules aren't quite adherent to that. But we should try to use it when adding new rules. So maybe, like, `unnecessary-comprehension-any-all`?

Reply:
(Otherwise, all looks good to me.)

Reply:
Sounds good, done!